test/expressions/generateExpressionTests.py

```diff
@@ -237,12 +238,12 @@ def save_tests_in_files(N_files, tests):
 
 def handle_function_list(functions_input, signatures):
     """
-    Handles list of functions, splitting elements between functions and signatures.
+    Handles list of functions, splitting elements between functions and signatures.  
```

Comment:
If in the end there are conflicts and you have to update this pull request, check out the old version of this file. No need to add spaces here.

If there are no conflicts, then we'll just get spaces.